gst/nnstreamer/tensor_filter/tensor_filter.c

```diff
@@ -965,6 +965,7 @@ _compare_tensors (GstTensorsInfo * info1, GstTensorsInfo * info2)
 {
   gchar null[] = "";
   gchar *result = null;
+  gchar *dimstr;
```

Comment:
Uninitialized pointer variable may make a problem.
What about allocating the NULL to that pointer?

Reply:
If it is problematic situtaion, -Wall -Werror should have generated build errors...